testutil/repo.go

```diff
@@ -7,13 +7,30 @@ import (
 	"os"
 	"testing"
 
+	"github.com/ipfs/go-ipfs/plugin/loader"
+
 	conf "github.com/ipfs/ipfs-ds-convert/config"
 
-	config "github.com/ipfs/go-ipfs/repo/config"
+	config "github.com/ipfs/go-ipfs-config"
 	fsrepo "github.com/ipfs/go-ipfs/repo/fsrepo"
 	_ "github.com/multiformats/go-multiaddr-dns"
 )
 
+func init() {
+	// Datastores are (by default preloaded) plugins
+
+	pl, err := loader.NewPluginLoader("")
```

Comment:
Should we do this in `OpenRepo` so we can load other plugins?

Reply:
I don't really know how well this tool would work with 'unsupported' datastores, will open a separate issue

Reply:
https://github.com/ipfs/ipfs-ds-convert/issues/17